packages/mobile/components/StrongholdUnlock.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { unlockStronghold } from '@core/profile'
+    import { Button, PasswordInput, Text, HTMLButtonType } from 'shared/components'
+
+    export let subtitle: string | undefined = undefined
+    export let busyMessage: string = ''
+
+    export let onSuccess: (..._: any[]) => Promise<unknown>
+    export let onCancel: () => unknown
+
+    let password: string
+    let error: string
+    let isBusy = false
+
+    async function handleSubmit(): Promise<void> {
+        try {
+            error = ''
+            isBusy = true
+            await unlockStronghold(password)
+            onSuccess && (await onSuccess())
+            isBusy = false
+        } catch (err) {
+            console.error(err)
+            error = localize(err?.message ?? err)
+            isBusy = false
+        }
+    }
+
+    function handleCancelClick(): void {
+        onCancel && onCancel()
+    }
+</script>
+
+<div class="mb-5">
```

Comment:
Should we event render the subtitle Text component if there is no subtitle? Maybe we can wrap this in an `#if` ?